shipyard_blueprints/snowflake/tests/test_auth.py

```diff
@@ -18,41 +20,54 @@ def conn_helper(client: SnowflakeClient) -> int:
         return 1
 
 
-def test_good_connection():
```

Comment:
adding the above condition will make it so you don't have to have code that has been commented out

Reply:
the line before test that need valid env file add
@pytest.mark.skipif(not env_exists, reason="No .env file found")